components/database/postgresql-12/Makefile

```diff
@@ -0,0 +1,120 @@
+#
+# This file and its contents are supplied under the terms of the
+# Common Development and Distribution License ("CDDL"). You may
+# only use this file in accordance with the terms of the CDDL.
+#
+# A full copy of the text of the CDDL should have accompanied this
+# source. A copy of the CDDL is also available via the Internet at
+# http://www.illumos.org/license/CDDL.
+#
+
+#
+# Copyright 2012-2013, EveryCity Ltd. All rights reserved.
+# Copyright 2019 Alexander Pyhalov
+# Copyright 2019 Michal Nowak
+#
+
+BUILD_BITS=32_and_64
+
+include ../../../make-rules/shared-macros.mk
+
+COMPONENT_NAME=			postgresql
+COMPONENT_MJR_VERSION=	12
+COMPONENT_MNR_VERSION=	0
+COMPONENT_VERSION=		$(COMPONENT_MJR_VERSION).$(COMPONENT_MNR_VERSION)
+COMPONENT_SRC=			$(COMPONENT_NAME)-$(COMPONENT_VERSION)
+COMPONENT_ARCHIVE=		$(COMPONENT_SRC).tar.bz2
+COMPONENT_ARCHIVE_HASH=	sha256:cda2397215f758b793f741c86be05468257b0e6bcb1a6113882ab5d0df0855c6
+COMPONENT_ARCHIVE_URL=	http://ftp.postgresql.org/pub/source/v$(COMPONENT_VERSION)/$(COMPONENT_ARCHIVE)
+COMPONENT_PROJECT_URL=	https://www.postgresql.org
+
+include $(WS_MAKE_RULES)/common.mk
+
+TEST_TARGET=TEST_64
```

Comment:
We put `TEST_TARGET` before `common.mk`, so the logic there can process it.

Reply:
test pass...